src/components/BrowserFilter/BrowserFilter.react.js

```diff
@@ -137,10 +161,18 @@ export default class BrowserFilter extends React.Component {
                 <Button
                   color="white"
                   primary={true}
-                  value="Apply these filters"
-                  width="245px"
+                  value={this.state.confirmName ? 'Confirm' : 'Save filters'}
```

Comment:
Could you make the `Save` button not highlighted by default? There should only be 1 highlighted (default) button in a UI dialog.